config/checkstyle_non_main_files_checks.xml

```diff
@@ -23,6 +23,12 @@
                value="^(?!(.*href=|.*http(s)?:|import |(.* )?package |.* files=|.*\.dtd)).{101,}$"/>
       <property name="message" value="Line should not be longer than 100 symbols"/>
     </module>
+    <module name="RegexpSingleline">
+        <property name="format" value="(mvn [^-])|(mvn \-[^e\-v])"/>
```

Comment:
I have a concern about adding `-` and `v` and that it could lead to false negatives. We are only looking for the 1 character and not even testing what comes after `--`.

I don't know all maven arguments, but here is some examples I went off of:
````
mvn test
mvn -e test
mvn -version
mvn --version
mvn --not-e
mvn -verbose no e
````

Can we do some type of lookahead check for this? Maybe: `(mvn [^-])|(mvn \-(?!v)(?!-v)[^e])`
This won't hamper performance either, will it?

Reply:
ok, this is better!

Reply:
done, this is better!